paper/project.tex

```diff
@@ -81,13 +81,13 @@ \subsection{Trusting ITPs to scale collaboration}
 
 
-\paragraph{\textbf{The non-Lean pieces:}} While \emph{Lean} can check proofs of theorems up to the limitations described above, a project of this scale involves the use of several non-\emph{Lean} tools. For example there are tools which extract the proven implications and anti-implications. There are tools which construct various visualisations. There are also metaprograms which call external automated theorem provers, and extract proof certificates from them to construct a \emph{Lean} proof out of them. In keeping with the garbage-in garbage-out principle, if these tools get spurious inputs and throw spurious outputs, \emph{Lean} can only tell us that the proofs are incorrect after the formal proofs have been translated to \emph{Lean}. It cannot, for instance, stop us from generating a large number of spurious conjectures that misguide contributors because of a simple index error in an array. Further, the continous integration scripts that run \emph{Lean} and check the \emph{Lean} code with external checkers are not formally verified. Such bugs can only be unconvered by empirical testing and user reports. This highlights a basic caveat when using interactive theorem provers. These tools check something highly specific, a proof, against a specification. Contributors are responsible for the correctness of everything else. This makes the role of organisers and maintainers especially important.
+\paragraph{\textbf{The non-Lean pieces:}} While \emph{Lean} can check proofs of theorems up to the limitations described above, a project of this scale involves the use of several non-\emph{Lean} tools. For example there are tools which extract the proven implications and anti-implications. There are tools which construct various visualisations. There are also metaprograms which call external automated theorem provers, and extract proof certificates from them to construct a \emph{Lean} proof out of them. In keeping with the garbage-in garbage-out principle, if these tools get spurious inputs and throw spurious outputs, \emph{Lean} can only tell us that the proofs are incorrect after the formal proofs have been translated to \emph{Lean}. It cannot, for instance, stop us from generating a large number of spurious conjectures that misguide contributors because of a simple index error in an array. Further, the continuous integration scripts that run \emph{Lean} and check the \emph{Lean} code with external checkers are not formally verified. Such bugs can only be uncovered by empirical testing and user reports. This highlights a basic caveat when using interactive theorem provers. These tools check something highly specific, a proof, against a specification. Contributors are responsible for the correctness of everything else. This makes the role of organisers and maintainers especially important.
```

Comment:
The diff view doesn't show it but this merely changes "unconvered" -> "uncovered".